skopt/optimizer/optimizer.py

```diff
@@ -166,6 +168,14 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
             warnings.warn("The 'auto' option for the acq_optimizer will be "
                           "removed in 0.4.")
             acq_optimizer = "lbfgs"
+
+        if not acq_optimizer == "sampling" and \
```

Comment:
I think it is all right, to support `acq_optimizer="auto"` over here where we set it automatically depending upon the base estimator though @glouppe generally frowns at stuff like that. Raising an error by default is not very pretty.